src/helpers/git.ts

```diff
@@ -0,0 +1,28 @@
+import simpleGit from 'simple-git'
+
+import {GIT_BRANCH, GIT_REPOSITORY_URL, GIT_SHA} from './tags'
+
+export const getGitMetadata = async () => {
+  try {
+    const git = simpleGit({
+      baseDir: process.cwd(),
+      binary: 'git',
```

Comment:
As the `junit upload` command relies on `git` to fetch repository metadata maybe it should appear as an optional dependency in the README of the command?

Reply:
yeah that's a good point 👍